eole/utils/scoring_utils.py

```diff
@@ -54,9 +54,17 @@ def translate(self, model, gpu_rank, step):
         # (take 'inference' field of config if exists?)
         # Set "default" translation options on empty cfgfile
         self.config.training.num_workers = 0
+        is_seq2seq = model.encoder is not None and model.decoder is not None
+        print("is_seq2seq", is_seq2seq)
+        if not is_seq2seq:
+            if "insert_mask_before_placeholder" in self.config.transforms:
+                self.response_patterns = self.config.transforms_configs.insert_mask_before_placeholder.response_patterns
+        else:
+            self.response_patterns = None
+
         predict_config = PredictConfig(
             model_path=["dummy"],
-            src=self.config.data["valid"].path_src,
+            src="dummy",
```

Comment:
not sure to understand, don't we lose the the valid src ?

Reply:
No, it is passed to the inference iterator below.